### ISSUE TITLE
[Feature Request] Recursive reload from a directory

### ISSUE THREAD

**Issue by zhouzhuojie** [Teamwork/reload#10]
Hi,

Wondering if reload can support recursive subdirectory from a directory.

https://github.com/Teamwork/reload/blob/ed43622194092571d65b220857b0ea5f814664ea/reload.go#L54-L55

**Comment by arp242** [comment]
Yeah, don't mind accepting patches for this. Maybe as a new `ListDirs(path string, cb func()) []Dir` function which scans a directory. You can then do `reload.Do(log.Print, cb, reload.ListDirs("./tpl", reloadTpl)...)`.
